packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -95,6 +95,11 @@ export class SDKClient {
     // populate with consensusnode requests via SDK
     constructor(clientMain: Client, logger: Logger, register: Registry) {
         this.clientMain = clientMain;
+
+        if (process.env.CONSENSUS_MAX_EXECUTION_TIME) {
+            this.clientMain = clientMain.setMaxExecutionTime(Number(process.env.CONSENSUS_MAX_EXECUTION_TIME));
```

Comment:
nit: add comment highlighting what setting this actually configures.
Is this how long the SDK will wait for a transaction response from consensus node GRPC endpoint or something else.
There's also a requestTimeout so it'd be good to be clear the aim here in case future configurations are needed